multi-cluster-spring-boot/.applier/params/registry-sa-prod-cluster

```diff
@@ -0,0 +1,2 @@
+NAMESPACE=multicluster-spring-boot-dev
```

Comment:
The namespace here should be pointed to `-prod`, otherwise applier breaks with this message:
```
TASK [openshift-applier : Apply OpenShift objects based on template with params for 'projects : create prod cluster credential'] **********************************************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "oc process  --local   -f  /home/esauer/src/container-pipelines/multi-cluster-spring-boot/.applier/inventory-prod/../templates/image-mirror-sa.yml -n multicluster-spring-boot-prod --param-file=/home/esauer/src/container-pipelines/multi-cluster-spring-boot/.applier/inventory-prod/../params/registry-sa-prod-cluster | oc apply -n multicluster-spring-boot-prod -f - ", "delta": "0:00:00.515876", "end": "2018-08-27 15:29:19.789586", "failed_when_result": true, "msg": "non-zero return code", "rc": 1, "start": "2018-08-27 15:29:19.273710", "stderr": "error: the namespace from the provided object \"multicluster-spring-boot-dev\" does not match the namespace \"multicluster-spring-boot-prod\". You must pass '--namespace=multicluster-spring-boot-dev' to perform this operation.", "stderr_lines": ["error: the namespace from the provided object \"multicluster-spring-boot-dev\" does not match the namespace \"multicluster-spring-boot-prod\". You must pass '--namespace=multicluster-spring-boot-dev' to perform this operation."], "stdout": "", "stdout_lines": []}
	to retry, use: --limit @/home/esauer/src/container-pipelines/multi-cluster-spring-boot/galaxy/openshift-applier/playbooks/openshift-cluster-seed.retry
```